docker-compose-dev.yml

```diff
@@ -352,13 +384,80 @@ services:
       - MEMCACHIER_USERNAME=user
       - MEMCACHIER_PASSWORD=password
 
+  #: Development tools
+  types-dashboard:
+    image: lelylanlab/types-dashboard-ng:latest
+    restart: always
+    expose:
+      - 80
+    depends_on:
+      - api-proxy
+      - people
+      - websockets
+    environment:
+      - VIRTUAL_HOST=types-dashboard
+      - PUBLIC_HOST=types-dashboard
+      - LISTEN_HOST=0.0.0.0
+      - PORT=80
+      - RACK_ENV=development
+      - RAILS_ENV=development
+      - NODE_ENV=development
+      - LELYLAN_API_PUBLIC_URL=api-proxy
+      - LELYLAN_PEOPLE_PUBLIC_URL=people
+      - LELYLAN_CLIENT_ID=""
+      - LELYLAN_TYPES_DASHBOARD_PUBLIC_URL=types-dashboard
+      - LELYLAN_DEVICES_DASHBOARD_PUBLIC_URL=devices-dashboard
+      - LELYLAN_WEBSOCKETS_PUBLIC_URL=websockets
+      - LELYLAN_DEV_CENTER_PUBLIC_URL=dev-center
+      - LELYLAN_NODES_PUBLIC_URL=nodes
+  devices-dashboard:
+    image: lelylanlab/devices-dashboard-ng:latest
```

Comment:
@Tabrizian This docker images not available because docker autobuild have a problem with device-directive-ng version in bower repository.
In this pull request: https://github.com/lelylan/device-directive-ng/pull/106, i change device-directive-ng version but bower don't update this package.